SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift

```diff
@@ -29,14 +29,15 @@ public class SYMKBrowseMapViewController: UIViewController {
     
     private var mapSelectionManager = SYMKMapMarkersManager<SYMKMapPin>()
     private var compassController = SYUICompassController(course: 0, autoHide: true)
+    private var recenterController = SYMKMapRecenterController()
     private var poiDetailDataSource: SYMKPoiDetailDataSource?
     private var poiDetailViewController: SYUIPoiDetailViewController?
     
     override public func loadView() {
         let browseView = SYMKBrowseMapView()
         browseView.setupCompass(compassController.compass)
-//        browseView.recenter.setup(with: SYUIActionButtonViewModel(title: "", icon: SygicIcon.positionLockIos, style: .secondary))
-        browseView.recenter.isHidden = !useRecenterButton
+        browseView.setupRecenter(recenterController.button)
+        recenterController.button.isHidden = !useRecenterButton
```

Comment:
toto je este na zvazenie ci nastavovat properties piamo cez referenciu na view alebo spravit nejaku settings vrstvu ktora sa injectne controlleru

Reply:
jo, do buducna urcite lepsie cez settings injectnute do controllera

Reply:
jj, to by som riesil potom ked budeme aj pridavat novy modul a budeme vediet co a ako :-)